erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -385,7 +398,18 @@ void WebRtcConnection::onTransportData(std::shared_ptr<DataPacket> packet, Trans
   if (getCurrentState() != CONN_READY) {
     return;
   }
-  media_stream_->onTransportData(packet, transport);
+  char* buf = packet->data;
+  RtpHeader *head = reinterpret_cast<RtpHeader*> (buf);
+  RtcpHeader *chead = reinterpret_cast<RtcpHeader*> (buf);
+  uint32_t ssrc = head->getSSRC();
+  if (chead->isRtcp()) {
+    ssrc = chead->getSourceSSRC();
```

Comment:
I think this won't work for sender reports. In that case, we need `chead->getSSRC()`